runtime/src/accounts_db.rs

```diff
@@ -11715,9 +11716,12 @@ pub mod tests {
         accounts.store_for_tests(1, &[(&pubkey1, &zero_lamport_account)]);
         accounts.store_for_tests(2, &[(&pubkey1, &zero_lamport_account)]);
         // Root all slots
-        accounts.add_root(0);
-        accounts.add_root(1);
-        accounts.add_root(2);
+        accounts.get_accounts_delta_hash(0);
```

Comment:
`get_accounts_delta_hash` calls added to match the normal way we add accounts. Also, this is required once we're using the write cache to get the pubkeys into the set of pubkeys clean looks at. There is some discrepancy between `dirty_stores` wrt write cache vs no write cache.